### ISSUE TITLE
The executing line highlighting should animate the scroll to line

### ISSUE THREAD

**Issue by dmwyatt** [CCExtractor/vardbg#2]
It's jarring and confusing, particularly when the next line is many lines away from the current line.

For example, in the animation in the README, when it scrolls to the `insertion_sort` function implementation, it's not immediately clear what happened.

A possibly better implementation would have some sort of animation showing the scroll to that point and the movement of the highlight to that point.

**Comment by kdrag0n** [comment]
I'm not so sure it's a good idea because it might be a bit distracting and/or out-of-place. For example, most terminals don't have smooth scrolling in order to keep all the text aligned and some editors behave the same way.

I'll look into it though, thanks for the suggestion.

**Comment by dmwyatt** [comment]
Yes, I agree that most terminals do not do this.  However, it seems like that might not be so relevant to a tool that is at least partly designed for learning.
